include/aoapplication.h

```diff
@@ -5,6 +5,13 @@
 #include "datatypes.h"
 #include "discord_rich_presence.h"
 
+#if defined(BASSAUDIO)
+#include "bass.h"
```

Comment:
do we really need to include this _here_

Reply:
yes, because include guards can suck my dick